CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 
 ### Internal
 * Using Core 10.3.3.
+* Updated to dotnetcore 3.1.
```

Comment:
This should say that only test projects were updated and also make a note that we're no longer testing .NET Core 2.0.